catalog/message/shape/catalog-request-message-shape.ttl

```diff
@@ -28,9 +28,12 @@ dspace_shapes:
 dspace_shapes:CatalogRequestMessageShape
 	a sh:NodeShape ;
 	sh:targetClass dspace:CatalogRequestMessage ;
+	rdfs:subClassOf dspace_shapes:CatalogShape ;
 	
 	sh:property [
 		a sh:PropertyShape ;
 		sh:path dspace:filter ;
+		sh:datatype xsd:string ;
```

Comment:
It might be a string, it might be an object in the end. Therefore, please remove the line.